Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs

```diff
@@ -95,24 +97,13 @@ static ShardMapUtils()
 
         /// <summary>
         /// Validate .Net runtime 4.6 if Active Directory Aughentication is specified in connection string.
+        /// Below call will throw if connection string specifies Active Directory authentication and library
+        /// is not running with .NET 4.6.
```

Comment:
4th commit looks good, please merge. Thanks! ☺

From: Apurva Sahasrabudhe [mailto:notifications@github.com]
Sent: Wednesday, May 4, 2016 1:04 AM
To: Azure/elastic-db-tools <elastic-db-tools@noreply.github.com>
Subject: Re: [Azure/elastic-db-tools] AD support (#124)


In Src/ElasticScale.Client/ShardManagement/ShardMap/ShardMapUtils.cs<https://github.com/Azure/elastic-db-tools/pull/124#discussion_r62001470>:

> @@ -95,24 +97,13 @@ static ShardMapUtils()

>

>          /// <summary>

>          /// Validate .Net runtime 4.6 if Active Directory Aughentication is specified in connection string.

> +        /// Below call will throw if connection string specifies Active Directory authentication and library

> +        /// is not running with .NET 4.6.

You are right. Removing this method and adding a devnote. Also removed unused IsActiveDirectoryAuthenticationSupported from ShardMapUtils.cs

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHub<https://github.com/Azure/elastic-db-tools/pull/124/files/1d8d14442517356a7094a1365e3600c7954767b0..0dca00b1c69e60f6e5a88ba3ec0e2df1f996414d#r62001470>